CONTRIBUTING.md

```diff
@@ -136,7 +136,7 @@ Sometimes, you have information about a variable/function (such as how and when
 let numConnections: number | null = null;
 ```
 
-Doc comments start with `/**` and end with `*/`. In VS Code, doc comments will show up when you hover your mouse over the variable/function name, anywhere it's used. If your information would be useful there, please put it in a doc comment.
+Doc comments start with `/**` and end with `*/`. In VS Code, doc comments will show up when you hover your mouse over the variable/function name anywhere it's used. If your information would be useful there, please put it in a doc comment.
```

Comment:
I don't know why you removed the comma here. There are some cases where you need a comma or can't have one, but in many cases, commas are optional. This is one of those cases, and the comma adds clarity which is why I had it in.

Reply:
```suggestion
Doc comments start with `/**` and end with `*/`. In VS Code, doc comments will show up when you hover your mouse over the variable/function name, anywhere it's used. If your information would be useful there, please put it in a doc comment.
```